mail-filter/rspamd/rspamd-3.0-r3.ebuild

```diff
@@ -92,6 +92,7 @@ src_configure() {
 		-DENABLE_HYPERSCAN=$(usex cpu_flags_x86_ssse3 ON OFF)
 		-DENABLE_JEMALLOC=$(usex jemalloc ON OFF)
 		-DENABLE_LUAJIT=$(usex lua_single_target_luajit ON OFF)
+		-DENABLE_LTO=$(usex lto ON OFF)
```

Comment:
Does this just control -flto or something more? If so, we try to avoid USE=lto (although that said, I personally like advertising if an upstream supports it or not).

Reply:
It seems to just control `-flto` plus it appends `-fno-fat-lto-objects` as well.

For `-DENABLE_LTO=ON` it calls:
```cmake
	check_ipo_supported (RESULT SUPPORT_LTO OUTPUT LTO_DIAG )
		if (SUPPORT_LTO)
			set (CMAKE_INTERPROCEDURAL_OPTIMIZATION TRUE)
```

Reply:
Hm, let's leave it out for now then (as you have).